client/my-sites/feature-upsell/redirect-if.jsx

```diff
@@ -33,26 +32,31 @@ export class UpsellRedirectWrapper extends React.Component {
 
 	getSnapshotBeforeUpdate() {
 		this.goToUpsellPageIfRequired();
+		return null;
 	}
 
+	componentDidUpdate() {}
```

Comment:
Do we need this?

Reply:
React emits warning if it's not in place - it's supposed to be there whenever getSnapshotBeforeUpdate is used

Reply:
@sirreal actually it made sense to use `getSnapshotBeforeUpdate ` at some point (as a close replacement of now deprecated `componentWillUpdate`), but when I think about it we can just use `componentDidUpdate` now - I just updated this PR